src/components/PopupBase/__tests__/PopupBase.spec.tsx

```diff
@@ -14,28 +16,32 @@ describe('<PopupBase> that adds basic anchor/popup functionality to rendered com
     const popupRendererMock = ({ setPopupVisibility }) => (
         <PopoverDummy setPopupVisibility={setPopupVisibility} />
     );
-    let wrapper;
+    let view: RenderResult;
 
     describe('rendering', () => {
         beforeEach(() => {
-            wrapper = mount(
+            view = render(
                 <PopupBase anchorRenderer={anchorRendererMock} popupRenderer={popupRendererMock} />
             );
         });
 
         it('should render with minimal props correctly', () => {
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('Popover')).toHaveLength(0);
+            expect(view.container).toMatchSnapshot();
+            expect(screen.queryByRole('group')).not.toBeInTheDocument();
         });
-        it('should render popup when requested', () => {
+
+        it('should render popup when requested', async () => {
+            const user = userEvent.setup();
             // trigger setPopupVisibility(true) through our dummy component
-            wrapper.find('button').simulate('click');
+            await user.click(screen.getByRole('button'));
 
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('Popover')).toHaveLength(1);
+            expect(view.container).toMatchSnapshot();
```

Comment:
the snapshot will not show anything popup related because it is a portal.

Reply:
:+1:

Reply:
I didn't mean to remove the snapshot. You can make it to appear in the snapshot if you use `view.baseElement` instead of the `view.container`. I think you added that to the TESTING_LIBRARY_MIGRATION.md file.

Reply:
I think we don't need it. The test case **should render popup when requested** and i just check it  expect(screen.getByRole('group')).toBeInTheDocument();

Reply:
:+1: